ansible/config_sbps_iscsi_targets.yml

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ansible-playbook
+#
+# MIT License
+#
+# (C) Copyright 2024 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+# Configure "Management_Worker" (all worker nodes) by default OR
+# configure only subset of worker nodes("Management_Worker") defined in 
+# CFS config/ HSM group, set by user/ admin during node personalization.
+#
+# Install and configure SBPS on iSCSI targets (worker NCNs)
+- hosts: Management_Worker
+  gather_facts: yes
+  any_errors_fatal: true
+  remote_user: root
+  vars_files:
+    - vars/csm_repos.yml
+    - vars/csm_packages.yml
+  roles:
+    # Apply k8s label on all the intended worker nodes
+    - role: csm.sbps.apply_label
+    # Install SBPS iSCSI LIO target + targetcli packages
+    # Note: Currently below RPMs are part of NCN node image. Below role was added for 
+    # convenience during dev/ test when image is not available/ ready.
+    #- role: csm.packages
+    #  vars:
+    #  packages: "{{ python-rtslib-fb-common + targetcli-fb-common + python3-targetcli-fb }}"
+    #  when: ansible_distribution_file_variety == "SUSE"
```

Comment:
We don't want to check in plays with commented sections like this. If we don't need the section, we should just remove it.